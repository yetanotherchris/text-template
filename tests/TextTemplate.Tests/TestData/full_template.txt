Dear {{ .Name }},
{{ if .Attended }}
It was a pleasure to see you.
{{ else }}
Sorry you couldn't make it.
{{ end }}
You brought: {{ for item in Items }}{{ item }},{{ end }}
Thank you for the lovely {{ .Gift }}.
