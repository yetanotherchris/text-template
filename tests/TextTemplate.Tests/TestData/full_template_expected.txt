Dear Bob,

Sorry you couldn't make it.

You brought: book,pen,
Thank you for the lovely toaster.
